site/content/en/docs/administration/advanced/webhooks.md

```diff
@@ -0,0 +1,356 @@
+---
+title: 'Webhooks'
+linkTitle: 'Webhooks'
+description: 'Instructions for working with CVAT Webhooks'
+---
+
+## Create Webhook
+
+In CVAT you can create webhook for project or for organization.
+For creation, you can use our user interface or direct API calls.
+
+In order to create webhook via an API call, see the swagger documentation. (TO-DO: ADD LINK HERE)
+And also see examples of creating webhooks in our REST API tests. (TO-DO: ADD LINK HERE)
+
+### Create Webhook for project
+
+To create webhook for CVAT project, follow the steps:
+
+`Project -> Actions -> Setup Webhooks`
+
+![](/images/create_project_webhook.gif)
+
+### Create Webhook for organization
+
+To create webhook for CVAT organization, follow the steps:
+
+`Organization -> Settings -> Actions -> Setup Webhooks`
+
+![](/images/create_organization_webhook.gif)
+
+## List of available events
+
+| Resource     | Create | Update | Delete |
+| :---:        | :----: | :----: | :----: |
+| Organization |        |   X    |        |
```

Comment:
Is it exist any ways to add emojis for markdown file?

Reply:
I think you can just copy-paste any unicode character, but need to make sure it will rendered correctly.
https://unicode-table.com/en/search/?q=check